yt_dlp/extractor/bundestag.py

```diff
@@ -0,0 +1,127 @@
+        'only_matching': True,
+    }, {
+        'url': 'http://dbtg.tv/fvid/3594346',
+        'only_matching': True,
+    }]
+
+    _OVERLAY_URL = 'https://www.bundestag.de/mediathekoverlay'
+    _INSTANCE_FORMAT = 'https://cldf-wzw-od.r53.cdn.tv1.eu/13014bundestagod/_definst_/13014bundestag/ondemand/3777parlamentsfernsehen/archiv/app144277506/145293313/{0}/{0}_playlist.smil/playlist.m3u8'
+
+    _SHARE_URL = 'https://webtv.bundestag.de/player/macros/_x_s-144277506/shareData.json?contentId='
+    _SHARE_AUDIO_REGEX = r'/\d+_(?P<codec>\w+)_(?P<bitrate>\d+)kb_(?P<channels>\w+)_(?P<lang>\w+)_\d+\.(?P<ext>\w+)'
+    _SHARE_VIDEO_REGEX = r'/\d+_(?P<codec>\w+)_(?P<width>\w+)_(?P<height>\w+)_(?P<bitrate>\d+)kb_\w+_(?P<lang>\w+)_\d+\.(?P<ext>\w+)'
+
+    def _bt_extract_share_formats(self, video_id):
+        share_data = self._download_json(
+            f'{self._SHARE_URL}{video_id}', video_id, note='Downloading share format JSON')
+        error = traverse_obj(share_data, ('status', 'code', {int}))
+        if error != 1:
+            self.report_warning(format_field(
+                share_data, [('status', 'message', {str})],
+                'Share API response: %s', default='Unknown Share API Error')
+                + bug_reports_message())
+            return
+
+        for name, url in share_data.items():
+            if not isinstance(name, str) or not name.startswith('audio') or not url_or_none(url):
+                continue
+
+            match = re.search(self._SHARE_AUDIO_REGEX, url)
+            yield {
+                'format_id': name,
+                'url': url,
+                'vcodec': 'none',
+                **traverse_obj(match, {
+                    'acodec': 'codec',
+                    'abr': ('bitrate', {int_or_none}),
+                    'ext': 'ext',
+                }),
+            }
+
+        for name, url in share_data.items():
+            if not isinstance(name, str) or not name.startswith('download') or not url_or_none(url):
+                continue
+
+            match = re.search(self._SHARE_VIDEO_REGEX, url)
+            yield {
+                'format_id': name,
+                'url': url,
+                **traverse_obj(match, {
+                    'vcodec': 'codec',
+                    'tbr': ('bitrate', {int_or_none}),
+                    'width': ('width', {int_or_none}),
+                    'height': ('height', {int_or_none}),
+                    'ext': 'ext',
+                }),
+            }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        formats = []
+        result = {'id': video_id, 'formats': formats}
+
+        try:
+            formats.extend(self._extract_m3u8_formats(self._INSTANCE_FORMAT.format(video_id), video_id))
+        except ExtractorError as error:
+            if isinstance(error.cause, HTTPError) and error.cause.status == 404:
+                raise ExtractorError('Could not find video id', expected=True) from None
```

Comment:
?
```suggestion
                raise ExtractorError('Could not find video id', expected=True)
```

Reply:
An artifact from before where I reextracted the instance url format, will remove